src/main/java/Duke.java

```diff
@@ -1,10 +1,190 @@
+                +" in the list");
+        printSeparator();
+        numOfTasks++;
+        return numOfTasks;
+    }
+
+    private static int commandToDo(Task[] currentTasks, int numOfTasks, String[] splitUserInput) {
+        currentTasks[numOfTasks] = new ToDo(splitUserInput[1]);
+        printSeparator();
+        System.out.println("Got it. I've added this task: \n" +"  " + addBrackets(currentTasks[numOfTasks].getType())
+                + addBrackets(currentTasks[numOfTasks].getStatusIcon()) + " "+ splitUserInput[1] + " \n" + "Now you have "
+                + (numOfTasks +1) + (numOfTasks == 0 ? " task" : " tasks" ) +" in the list");
+        numOfTasks++;
+        printSeparator();
+        return numOfTasks;
+    }
+
+    private static int commandEvent(Task[] currentTasks, int numOfTasks, String s) {
+        String[] splitInfoAndDate = s.trim().split("/at",2 );
+        currentTasks[numOfTasks] = new Event(splitInfoAndDate[0], splitInfoAndDate[1] );
+        printSeparator();
+        System.out.println("Got it. I've added this task: \n" + "  " + addBrackets(currentTasks[numOfTasks].getType())
+                + addBrackets(currentTasks[numOfTasks].getStatusIcon()) +" " + splitInfoAndDate[0] + " (at: " +
+                splitInfoAndDate[1] +  ")\n" + "Now you have " + (numOfTasks +1) + (numOfTasks == 0 ? " task" : " tasks" )
+                +" in the list");
+        numOfTasks++;
+        printSeparator();
+        return numOfTasks;
+    }
+
+    //List of valid commands
+    private static void commandHelp(){
+        printSeparator();
+        System.out.println("Here are the range of commands:\n 1.todo\n 2.deadline\n 3.event\n 4.list\n 5.done\n 6.bye\n ");
+        printSeparator();
+    }
+
+
+    /*
+     * ===========================================
+     *           COMMAND LOGIC
+     * ===========================================
+     */
+    private static void showWelcomeMessage() {
+        printSeparator();
+        String logo = "           ____          ____       \n"
+                + " __(\")__  |  _ \\ _   _  |  _ \\  ____ \n"
+                + "(__   __) | | | | | | | | | | || ___\\\n"
+                + "  /   \\   | |_| | |_| | | |_| |\\  __/\n"
+                + " (_/ \\_)  |____/ \\__,_| \\____/  \\____|\n";
+        System.out.println(logo);
+        System.out.println("Sup! I'm Dude!\nWhat can a brother do for you?");
+        printSeparator();
+    }
+    //Function to encapsulate description with square brackets
+    private static String addBrackets(String description){
+        String s;
+        s = "[" + description +"]";
+        return s;
+    }
+    //Function overload of addBrackets
+    private static String addBrackets(char description){
+        String s;
+        s = "[" + description +"]";
+        return s;
+    }
+
+    //Command "bye" is given; Exit program
+    private static void exitProgram(){
```

Comment:
I think the naming of the method can be improved, as the exit program would suggest the method ran System.exit(0);